lib/superobject/superobject.pas

```diff
@@ -142,7 +142,7 @@ interface
   SOIChar = Word;
   PSOChar = PChar;
   SOString = string;
-{$ifend}
+{$ENDIF}
```

Comment:
I think that this will break on version < X4. See: http://docwiki.embarcadero.com/RADStudio/Tokyo/en/Legacy_IFEND_(Delphi)

Reply:
That is interesting.  `{$endif}` works fine in my old Delphi 3 code, `{$ifend}` is alien.